### PR TITLE
fix: move RUNNER_START_DELAY to kong

### PR DESCRIPTION
**Pull request by stuartwdouglas**

### CODE REVIEW

File: backend/runner/runner.go [hunk 1]
```diff
@@ -52,6 +51,7 @@ type Config struct {
 	Language              []string        `short:"l" help:"Languages the runner supports." env:"FTL_LANGUAGE" default:"go,kotlin,rust"`
 	HeartbeatPeriod       time.Duration   `help:"Minimum period between heartbeats." default:"3s"`
 	HeartbeatJitter       time.Duration   `help:"Jitter to add to heartbeat period." default:"2s"`
+	RunnerStartDelay      int             `help:"Time in seconds for a runner to wait before contacting the controller. This can be needed in istio environments to work around initialization races." env:"FTL_RUNNER_START_DELAY" default:"0"`
```
Comment (alecthomas) [inline backend/runner/runner.go:54]: This should be a `time.Duration` - it will automatically parse values like `1s`, `1m`, etc.

Comment (alecthomas) [reply backend/runner/runner.go:54]: (we'll need to update any locations where FTL_RUNNER_START_DELAY is set to an unadorned integer value though, but it's better for this to be correct now than have to fix it later)

Comment (stuartwdouglas) [reply backend/runner/runner.go]: @alec-brooks I have updated to PR to fix this, but we will need to update the infra repo once this change is deployed.

**Review by alecthomas — APPROVED** [review]

**Comment by alecthomas** [comment]
Sorry! Forgot to approve!

**Pull request merged**